src/store/modules/user/actions.ts

```diff
@@ -31,7 +31,9 @@ const actions: ActionTree<UserState, RootState> = {
       const permissionId = process.env.VUE_APP_PERMISSION_ID;
       // Prepare permissions list
       const serverPermissionsFromRules = getServerPermissionsFromRules();
-      if (permissionId) serverPermissionsFromRules.push(permissionId);
+      
+      const removeDuplicatePermissionIds = new Set(serverPermissionsFromRules)
+      removeDuplicatePermissionIds.add(permissionId);
 
       const serverPermissions = await UserService.getUserPermissions({
         permissionIds: serverPermissionsFromRules
```

Comment:
We can remove the above code to create set.
```suggestion
        permissionIds: [...new Set(serverPermissionsFromRules)]
```